test/e2e/common.go

```diff
@@ -175,7 +175,7 @@ func EnsureImage(k8sVersion string) (imageURL string, imageChecksum string) {
 // DownloadFile will download a url and store it in local filepath.
 func DownloadFile(filePath string, url string) error {
 	// Get the data
-	resp, err := http.Get(url) //nolint:noctx
+	resp, err := http.Get(url) //nolint:noctx #nosec
```

Comment:
`#nosec` here is not doign anything (not like this nor as `//#nosec`, and if moved to start of comment, then `nolint` has no effect and golangci-lint fails. How to make both work on the same line?

Reply:
We may have to put one of them above. Something like 
```go
/* #nosec */
resp, err := http.Get(url) //nolint:noctx
```

Reply:
or //nolint:noctx,gosec

Reply:
Used @lentzi90's suggestion in the end.